package/kedro_viz/models/graph.py

```diff
@@ -528,6 +526,22 @@ def __post_init__(self, data_node: DataNode):
         if not data_node.is_free_input:
             self.run_command = f'kedro run --to-outputs="{data_node.full_name}"'
 
+    @staticmethod
+    def load_latest_metrics_data(dataset: Any) -> Optional[Dict[Any, Any]]:
+        """Load data for latest versions of the metrics dataset
+        Args:
+            dataset: the latest version of the metrics dataset

```

Comment:
I'm a little bit confused about what the input is here. Is it the actual dataset or just a version? And could we make the type more specific, it should only ever be `MetricsDataSet` that gets processed here right? For me personally, that already makes it clearer what this method does.

Reply:
You are right!! Initially I had it as MetricsDataSet but it kept failing circleCI then I tried doing forward reference "Metrics Dataset" but that also didn't work so I change it to Any as a last resort just so it could pass circleCI. 
I can share with you the fail circleCI error if u have time to help investigate

Reply:
Fixed